### PR TITLE
added meta

### PR DESCRIPTION
**Pull request by daneyul**
I think the meta's been added properly? Not sure if the og:title and twitter:title is working as it's supposed to. I referenced the vue-meta docs.

### CODE REVIEW

**Comment by danrocha** [comment]
What is happening to make you think og: properties are not working?

File: src/App.vue [hunk 1]
```diff
@@ -23,6 +23,58 @@ export default {
     title: 'Welcome',
     // all titles will be injected into this template
     titleTemplate: '%s | aec.works',
+    htmlAttrs: {
+      lang: 'en',
+      amp: false,
+    },
+    meta: [
+      { charset: 'utf-8' },
+      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
```
Comment (gtalarico) [inline src/App.vue:32]: Do these tags clash with tags currently defined in `public/index.html`

File: src/App.vue [hunk 2]
```diff
@@ -23,6 +23,58 @@ export default {
...
+        template: (chunk) => `${chunk} | aec.works`,
+      },
+      {
+        property: 'twitter:description',
+        content:
+          'An open-source AEC community platform that promotes inclusive and enriching conversations through content-sharing and discussions.',
+      },
+      {
+        property: 'twitter:image',
+        content: './assets/images/meta.png',
```
Comment (gtalarico) [inline src/App.vue:75]: Is this working as an absolute url path?
I thought you might have to use `require('@/assets/...`)` to get vue compiler to pick it up from assets folder and process

**Review by gtalarico — COMMENTED** [review]
Good sutff @daneyul !

File: src/App.vue (hunk 1 above)
Comment (daneyul) [reply src/App.vue:32]: Ah yeah good point. I'll remove it from the vue-meta and leave the ones in the index.html

File: src/App.vue (hunk 2 above)
Comment (daneyul) [reply src/App.vue:75]: oh yeah, I guess it would be the same for vue. makes sense.

**Comment by daneyul** [comment]
> What is happening to make you think og: properties are not working?

I'm using `template: (chunk) => `${chunk} | aec.works`` but I'm not sure if it's dynamically pulling in the right data. Just copied and pasted from the docs. I noticed above that the title template is set as `titleTemplate: '%s | aec.works',` but the meta data supposedly uses a different code. Maybe it looks different on build than local?

**Comment by daneyul** [comment]
added a new commit with above changes

**Comment by gtalarico** [comment]
| Maybe it looks different on build than local?
we can merge it as is and fix it later if meta is incorrect

**Review by danrocha — APPROVED** [review]
looks good to me

File: src/App.vue [hunk 3]
```diff
@@ -23,6 +23,56 @@ export default {
...
+      },
+      {
+        property: 'twitter:title',
+        vmid: 'twitter:title',
+        template: (chunk) => `${chunk} | aec.works`,
+      },
+      {
+        property: 'twitter:description',
+        content:
+          'An open-source AEC community platform that promotes inclusive and enriching conversations through content-sharing and discussions.',
```
Comment (gtalarico) [inline src/App.vue:69]: thinking of "stronger" title... 🤔 
`An Open Source Community Platform for the AEC Industry`
or `The Open Source AEC Community Platform`
Thoughts?

File: src/App.vue [hunk 4]
```diff
@@ -23,6 +23,56 @@ export default {
...
+      },
+      {
+        property: 'twitter:image',
+        content: require('@/assets/images/meta.png'),
```
Comment (gtalarico) [inline src/App.vue:73]: I haven't been too consistent with this, 
so we have a few variants of the logo. I will see if I can create a new figma page with "final" ones. I think I prefer the slightly thicker one... What do we think?

image
vs
image

File: yarn.lock [hunk 5]
```diff
@@ -9842,6 +9860,13 @@ vue-meta@^2.4.0:
   dependencies:
     deepmerge "^4.2.2"
 
+vue-router-sitemap@0.0.4:
```
Comment (gtalarico) [inline yarn.lock:9863]: is this being used?
curious why  `packages.json` was not updated as well in this PR.
was it?

**Review by gtalarico — COMMENTED** [review]
looks like we have a merge conflict with yarn lock 🤷

File: src/App.vue (hunk 3 above)
Comment (daneyul) [reply src/App.vue:69]: I like “The Open Source...” sounds legit

File: src/App.vue (hunk 4 above)
Comment (daneyul) [reply src/App.vue:73]: Oh yeah, the thicker border with the the yellow looks good and consistent with other brand assets

File: yarn.lock (hunk 5 above)
Comment (daneyul) [reply yarn.lock:9863]: Oh no it’s not being used. Thought i got rid of that

File: src/App.vue (hunk 4 above)
Comment (gtalarico) [reply src/App.vue:73]: That 'W' alignment though 🤦
I will clean these bran assets up 
We can use any of now and will update later

**Comment by gtalarico** [comment]
* Cleaned merge conflict 
* updated logo asset
* added placeholder sitemap
* allow robots, except people page

File: yarn.lock (hunk 5 above)
Comment (gtalarico) [reply yarn.lock]: ok rebuild lock file

**Pull request merged**